packages/core/src/odata-common/request/odata-request.ts

```diff
@@ -55,7 +55,9 @@ export class ODataRequest<RequestConfigT extends ODataRequestConfig> {
    * @returns The absolute URL for the request
    */
   url(): string {
-    return `${removeTrailingSlashes(this.resourceUrl())}${this.query()}`;
+    return `${removeTrailingSlashes(this.resourceUrl())}${removeTrailingSlashes(
```

Comment:
Don't you need a slash in between the resource url and the appended path?

Reply:
Please also add a test for this.

Reply:
Here I have a [test](https://github.com/SAP/cloud-sdk-js/pull/1276/files#diff-6cf9bb4dc4da9e788a878f6ee227258a0b6b3d5e44e2057fd85f0a8bdf32f950R22).

Reply:
> Don't you need a slash in between the resource url and the appended path?

This is about whether the user should provide `/` as part of the parameter, currently here is one example: 
`.appendPath('/to_MultiLink')`

Reply:
I would then make this test as simple as possible and only check the URL from `url`/`executeRaw`.

Reply:
As discussed with @FrankEssenberger , based on the reasons below, the user needs to provide all the appended path:
- if the SDK adds the missing `/` in the beginning, it cannot handle use case when not `/` is needed
- as a workaround API, we aim to provide a powerful API instead of an intelligent solution
- also having everything from the parameter provided by the user is explicit.

Reply:
Almost convinced, still:
* Do you have an example of a use case, where the leading `/` is not needed?
* Isn't this inconsistent with the fact that we are removing trailing slashes? Should we consider to not remove trailing slashes? (See next question)
* What happens if someone uses `.appendPath('/')`? Currently it will basically be ignored, but should it?

Reply:
As discussed, we remove the `removeTrailingSlashes` for appended path so users have the full power to control the url.